lib/Doctrine/Common/DataFixtures/ReferenceRepository.php

```diff
@@ -234,6 +312,16 @@ public function getReferences()
         return $this->references;
     }
 
+    /**
+     * Get all stored references
+     *
+     * @psalm-return array<class-string<array<string, object>>
```

Comment:
this is wrong. `class-string<array<string, object>` does not make sense as a type.

Reply:
I fixed the typo